pom.xml

```diff
@@ -12,9 +12,19 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
+                <groupId>com.google.inject</groupId>
+                <artifactId>guice</artifactId>
+                <version>4.0</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.inject.extensions</groupId>
+                <artifactId>guice-multibindings</artifactId>
```

Comment:
This dep needs to be explicitly inherited by modules that use it. Right now it's an indirect dependency through the spoofax dependency, which is why the build still works.